docs/_templates/launch.html

```diff
@@ -13,7 +13,7 @@
 </a>
 
 <a
-  href="{{ pathto('_static/retro/index.html', 1) }}"
+  href="{{ pathto('_static/retro/', 1) }}?path=notebooks/intro-retro.ipynb"
```

Comment:
Looks like removing the `index.html` here creates an infinite redirect loop?

https://user-images.githubusercontent.com/591645/149344871-68ecb47f-2b17-4d43-a8e0-fa2a6e367966.mp4

Reply:
yeah, different hosts react differently with respect to `index.html`, trailing slashes, etc. and `index.html` is the only one that's going to _work_ work. That's why the rediraffe stuff in the first place for "pretty" urls.

Reply:
```suggestion
  href="{{ pathto('_static/retro/index.html', 1) }}?path=notebooks/intro-retro.ipynb"
```